robot/Cumulus/tests/browser/zpayment_allocations/enable_pmt_alloc_and_test.robot

```diff
@@ -12,49 +15,72 @@ Suite Teardown  Run keywords
+    Current Page Should Be                 Details                    npe01__OppPayment__c
+    Select Tab                             Related
     Verify Payment Allocations    
     ...    &{def_gau}[Name]=$100.00
-    Click Wrapper Related List Button    Payment Allocations    New
-    Populate Lookup Field    General Accounting Unit    &{gau}[Name]
-    Populate Field    Amount    40
-    Click Modal Button    Save
+    Click Related List Button              Payment Allocations        New
+    Wait For Modal                         New                        GAU Allocation
+    Populate Lookup Field                  General Accounting Unit    &{gau}[Name]
+    Populate Field                         Amount                     40
+    Click Modal Button                     Save
     Wait Until Modal Is Closed
     Verify Payment Allocations    
     ...    &{def_gau}[Name]=$60.00
     ...    &{gau}[Name]=$40.00     
-    Select Tab    Details
-    Click Link    &{opportunity}[Name]
-    Select Tab    Related
-    Verify Allocations    GAU Allocations
+    Select Tab                             Details
+    Click Link                             ${data}[contact1_opportunity][Name]
+    Current Page Should Be                 Details                    Opportunity
+    Select Tab                             Related
+    Verify Allocations                     GAU Allocations
     ...    &{def_gau}[Name]=$60.00
     ...    &{gau}[Name]=$40.00
 
 Update GAU Allocations and Verify Payment Allocations Sync
-    [tags]    unstable
-    Go To Page    Details    Opportunity    object_id=&{opportunity}[Id]
-    Select Tab    Related
-    Click Special Related List Button  GAU Allocations    Manage Allocations
-    Choose Frame    Manage Allocations
-    Add GAU Allocation    Percent 0    60
-    Click Button    Save
-    Unselect Frame
-    Verify Allocations    GAU Allocations
+    [Documentation]                        Once Allocations are Enabled and a Default GAU is added, create an opportunity via API.
+    ...                                    Open Opportunity and Manage Allocations from GAU Allocations related list.
+    ...                                    Add a new Allocation to another GAU and verify that the Allocation is split between default and new GAU.
+    ...                                    Open Payments from related list and verify that same allocation is reflected under Payment Allocations related list.
+    [tags]                                 W-039821                   feature:Payment Allocations
+    Go To Page                             Detail
+    ...                                    Opportunity
+    ...                                    object_id=${data}[contact2_opportunity][Id]
+    Select Tab                             Related
+    Click Special Related List Button      GAU Allocations            Manage Allocations
+    Wait For Page Object                   Custom                     ManageAllocations   
```

Comment:
Are Manage Allocations in line 70 & 71 the same or different?

Reply:
line 70 is button name, 71 is page object name